dicom-archive/DICOM/DCMSUM.pm

```diff
@@ -1112,10 +1112,12 @@ sub trimwhitespace {
 
 =head3 date_format($first, $second)
 
-Pass it a date in YYYYMMDD and you get YYYY-MM-DD.
-Pass it two of these and you get the difference in decimal and Y M +/- Days.
+If only one date argument is provided, then it will convert YYYYMMDD date
+format into YYYY-MM-DD.
+If two date arguments are provided, thenit will compute the difference in
```

Comment:
missing space between `then` and `it`
(I think I got your bug @cmadjar ):)

Reply:
lol! @MounaSafiHarab should be fixed now. Thanks!